a/amazon-metadata.yml

```diff
@@ -0,0 +1,19 @@
+amazon-metadata:
+  image: rancher/os-amazon-metadata:v0.9.2${SUFFIX}
```

Comment:
Hmm... OK, what versioning scheme should be preferred? I mean, what tag should I use?